packages/easy-hydrator/config/config.php

```diff
@@ -0,0 +1,25 @@
+<?php declare(strict_types=1);
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Symfony\Component\Cache\Adapter\FilesystemAdapter;
+use Symplify\EasyHydrator\ArrayToValueObjectHydrator;
+use Symplify\PackageBuilder\Strings\StringFormatConverter;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $services = $containerConfigurator->services();
+
+    $services->defaults()
+        ->public()
+        ->autowire()
+        ->autoconfigure();
+
+    $services->load('Symplify\\EasyHydrator\\', __DIR__ . '/../src')
+        ->exclude([__DIR__ . '/../src/DepencencyInjection/*', __DIR__ . '/../src/EasyHydratorBundle.php']);
+
+    $services->set(FilesystemAdapter::class);
+
+    $services->set(StringFormatConverter::class);
+
+    $services->set(ArrayToValueObjectHydrator::class);
```

Comment:
Why does this show up here, when it wasn't present in the yaml file?

Reply:
That's left over from some debugging. Should be removed, thanks

Reply:
https://github.com/symplify/symplify/commit/53489df2e08189aca73a46ac5bcab81530084b7f